test/test_um2netcdf.py

```diff
@@ -12,6 +14,9 @@
 import iris.cube
 import iris.coords
 
+D_LAT_N96 = 1.25
+D_LON_N96 = 1.875
```

Comment:
Could you comment what these numbers are for clarity?

Reply:
Are these meant to be `DELTA_LAT_N96` & `DELTA_LON_N96` ?

Reply:
I tried to use similar naming here as in other parts of `um2nc`, eg [here](https://github.com/ACCESS-NRI/um2nc-standalone/blob/92096ce86f3cdd305570e940397b06633ae8777b/umpost/um2netcdf.py#L390) and [here](https://github.com/ACCESS-NRI/um2nc-standalone/blob/92096ce86f3cdd305570e940397b06633ae8777b/umpost/um2netcdf.py#L364)

I'm wondering would it be best to stick to that naming, and add a comment clarifying that these are grid spacings in degrees?

Reply:
Yes just go the comment.

Reply:
Comments added in 21dc396